pkg/windows/windows.go

```diff
@@ -735,13 +734,19 @@ func (vm *windows) removeDirectories() error {
 }
 
 // transferFiles copies various files required for configuring the Windows node, to the VM.
-func (vm *windows) transferFiles() error {
+func (vm *windows) transferFiles(tlsData map[string][]byte) error {
```

Comment:
Rather than making this a special case this should be handled in the same way the other files are. It should be part of `filesToTransfer`, which is generated when the Windows object is created. Doing it this way is adding additional arguments to any function which needs to call `transferFiles`, and is moving away from the established pattern.

Reply:
If this doesn't make sense for this file, which may be the case, then it should be not be handled by the `transferFiles` function.

Reply:
Yes looking at this more it seems like this shouldn't be handled here, rather in the createBootstrapFiles function https://github.com/openshift/windows-machine-config-operator/blob/0531500ee3ef8e88fabed3e05087338f733164e6/pkg/nodeconfig/nodeconfig.go#L344